app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  def create
+    authenticate_user!
+    @program = Program.find(params[:program_id])
+    @user = current_user
+    @review = Review.new(review_params)
+    @review.program = @program
```

Comment:
Look into using `.merge` in your `review_params` so that you don't need to set the user and associated program each time.